File: processing-py/boids-sim/sketch.py
from vv_trigo import Vector2d
from boids import Boid

boid = None


def setup():
    size(600, 600)
    noStroke()
    global boid
    boid = Boid(width / 2, height / 2, 10, 1)


def draw():
    background(255)

    mouse = Vector2d(mouseX, mouseY)

    #look_at_mouse = Vector2d.subtract(vec1=mouse, vec2=boid.pos)
    # look_at_mouse.normalize()
    # look_at_mouse.mult(30)

    boid.arrive(mouse)

    # boid.apply_force(look_at_mouse)
    boid.wrap_edges(width, height)
    boid.update()

    fill(0)
    boid.render()

File: p5js/kadenze-nature-of-code/3-03/sketch.js
var vehicle;

function setup() {
  createCanvas(640, 360);
  vehicle = new Vehicle(width/2, height/2, 8, 2);
}

function draw() {
  background(51);
  
  var target = createVector(mouseX, mouseY);
  vehicle.seek(target);
  
  vehicle.update();
  vehicle.render();
}